Makefile

```diff
@@ -26,12 +26,9 @@ install-onnx-dev: install-onnx-test install-pre-commit install-pylint
 test-onnx:
 	pytest tests/onnx $(DATA_ARG) --junitxml ${JUNITXML_PATH}
 
-ONNX_PYFILES := $(shell find examples/post_training_quantization/onnx -type f -name "*.py")
+ONNX_PYFILES := $(shell find -path "*onnx*.py")
```

Comment:
It will contains all file include untracked by git (venv, build, not added to git files).
Need use git tools like `git ls-files | grep -P ".*onnx.*py$"` but need to check that it's not file for torch like `tests/torch/quantization/test_onnx_export.py`

Reply:
This problem is not applicable for CI pylinting. Is it real case when  a developer has additional files in repo that are not tracked by git? Is it big problem if Pylint will be checking not tracked files?

regarding the  tests/torch/quantization/test_onnx_export.py - I believe there is no problem in a way pylint working now: it's checking this file both by onnx and torch, and it looks ok

Reply:
In my case, pylint checks files from virtual environment in direcory `venv` and `build` directory. 
And i always see errors from my tests scripts which will not be pushed to repo.


About double check, yes it's not problem now. But if new file will check before install required modules, we will get `import-error`.
You can simply avoid it by adding `-v or --invert-match` filter as you add for common files `git ls-files | grep -P ".*onnx.*py$" | grep -v -P ".*torch.*" ` 

For common it will looks like: 
`git ls-files | grep -P ".*py$" | grep -v -P ".*(torch|tensorflow|onnx|openvino).*" | grep -v -P "^tools"`


And where should check files from `tools`?

Reply:
I thought we shouldn't check tool directory because previously we never did that. Should we pylint tools directory? CC: @vshampor, @alexsu52

Reply:
Migrated to grep, @AlexanderDokuchaev, please check